default.nix

```diff
@@ -188,7 +188,7 @@ nightlyToolchains.${v} // rec {
       '';
     in
     pkgs.vscode-utils.buildVscodeExtension {
-      name = "rust-analyzer-${rust-analyzer-rev}";
+      pname = "rust-analyzer-${rust-analyzer-rev}";
       version = rust-analyzer-rev;
```

Comment:
Leaving the rev here is definitely wrong as it will be duplicated in the final name.

Reply:
I see. I'll close this, in any event, as it appears #184 is passing CI.